src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,19 @@
+export default class PokemonClient {
+  constructor() {}
+
+  async pokemonFetcherById(pokemon) {
+    const pokemons = await Promise.all(
+      pokemon.map(async (id) => {
+        const response = await fetch(`https://pokeapi.co/api/v2/pokemon/${id}`);
+        if (response.ok) {
+          return response.json();
```

Comment:
response.json is an async method, would change it to
`return await response.json()` even if it works this way